search-service/src/test/kotlin/com/egm/stellio/search/service/AttributeInstanceServiceTests.kt

```diff
@@ -84,8 +94,10 @@ class AttributeInstanceServiceTests : TimescaleBasedTests() {
 
         (1..10).forEach { _ -> attributeInstanceService.create(gimmeAttributeInstance()).block() }
 
-        val temporalQuery = TemporalQuery(emptyList(), TemporalQuery.Timerel.AFTER, Instant.now().atZone(ZoneOffset.UTC).minusHours(1),
-            null, null, null)
+        val temporalQuery = TemporalQuery(
```

Comment:
Strange, it is not consistent with formatting at https://github.com/stellio-hub/stellio-context-broker/pull/59/files#diff-8adafc7784840c932f2aeead03149087R69

Reply:
Yes indeed this is an error, I guess the forced newline made the compiler fail